introduction.md

```diff
@@ -0,0 +1 @@
+Hey there everyone, Alex here. I am a developer from Florida who is currently taking class at the University of Florida. Learning GitHub as we speak. It is cool how committing changes in this file to my branch automatically updates the pull request.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.